README.md

```diff
@@ -28,6 +28,22 @@ jobs:
 > [!NOTE]  
 > When authenticated access is required to publish a Build Scan®, it is recommended to provide as input `develocity-access-key` to the `maven-setup` step. This triggers a request for a [short-lived access token](https://docs.gradle.com/develocity/api-manual/#short_lived_access_tokens) instead of relying on the `DEVELOCITY_ACCESS_KEY` environment variable.
 
+If one wants to inject Develocity Maven Extension (and optionally CCUD Maven Extension), the `develocity-injection-enabled` input should be set to `true`. The `develocity-url`, `develocity-maven-extension-version`, and optionally `develocity-ccud-maven-extension-version` inputs should be set accordingly:
```

Comment:
I would enclose the injection-related doc in the README in a dedicated section. Feel free to adjust or not.

Reply:
I've updated the doc a bit, but I think some general rework of the README structure may be needed once all features are there.